apps/desktop-wallet/src/App.tsx

```diff
@@ -34,15 +35,20 @@ import { WalletConnectContextProvider } from '@/contexts/walletconnect'
 import { useAppDispatch, useAppSelector } from '@/hooks/redux'
 import UpdateWalletModal from '@/modals/UpdateWalletModal'
 import Router from '@/routes'
-import { syncAddressesData, syncAddressesHistoricBalances } from '@/storage/addresses/addressesActions'
-import { makeSelectAddressesUnknownTokens, selectAddressIds } from '@/storage/addresses/addressesSelectors'
-import { syncNetworkTokensInfo, syncUnknownTokensInfo } from '@/storage/assets/assetsActions'
-import { selectIsTokensMetadataUninitialized } from '@/storage/assets/assetsSelectors'
+import { syncAddressesAlphHistoricBalances, syncAddressesData } from '@/storage/addresses/addressesActions'
+import {
+  makeSelectAddressesUnknownTokens,
+  selectAddressIds,
+  selectAllAddressVerifiedFungibleTokenSymbols
+} from '@/storage/addresses/addressesSelectors'
+import { syncUnknownTokensInfo, syncVerifiedFungibleTokens } from '@/storage/assets/assetsActions'
+import { selectAreVerifiedFungibleTokensInitialized } from '@/storage/assets/assetsSelectors'
```

Comment:
FYI, you'll notice **many** renames. It took me half a day to wrap my head around the code and another full day to make this PR feature complete. Renaming things helped me clarify our code.

Reply:
Yeah I like it!